app/templates/docs/action.html

```diff
@@ -1,615 +1,643 @@
+    <ul class="simple">
+      <li><strong>Required permissions:</strong> read</li>
+      <li><strong>Arguments</strong>
+        <ul>
+          <li><code class="docutils literal"><span class="pre">field</span></code> (required)
+            <ul>
+              <li><strong>Valid Values:</strong> One of:
+                <ul>
+                  <li><code class="docutils literal"><span class="pre">closed</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idBoard</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">name</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">pos</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">subscribed</span></code></li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </div>
+  <div class="section" id="get-1-actions-idaction-member">
+    <h2>GET <span class="target" id="actions-idaction-member">/1/actions/[idAction]/member</span><a class="headerlink" href="#get-1-actions-idaction-member" title="Permalink to this headline"></a></h2>
+    <ul class="simple">
+      <li><strong>Required permissions:</strong> read</li>
+      <li><strong>Arguments</strong>
+        <ul>
+          <li><code class="docutils literal"><span class="pre">fields</span></code> (optional)
+            <ul>
+              <li><strong>Default:</strong> <code class="docutils literal"><span class="pre">all</span></code></li>
+              <li><strong>Valid Values:</strong> <code class="docutils literal"><span class="pre">all</span></code> or a comma-separated list of:
+                <ul>
+                  <li><code class="docutils literal"><span class="pre">avatarHash</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">avatarSource</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">bio</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">bioData</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">confirmed</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">email</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">fullName</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">gravatarHash</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idBoards</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idBoardsPinned</span></code></li>
+                  <li><code class="docutils literal"><span class="pre">idOrganizations</span></code></li>
+                  <li><code class="docutils 
```

Comment:
Same comment as above. Does this just work for `commentCard`? (if it even works for that)